### ISSUE TITLE
Tower of Hanoi (Recursive,Non-Recursive)

### ISSUE THREAD

**Issue by LikhithaTadikonda** [manan025/DS-Algo-Zone#113]
## 🚀 Feature
Tower of Hanoi Algo. Using Recursive and Non-Recursive 3 discs
(Write down a clear and concise description of what the feature is.)

### Have you read the Contribution Guidelines?
(Yes.)

## Pitch
Since it is asked in a lot of interviews. It is important to know about it and  I have codes working up to 3 disc.
Assign this to me for C.

## Assignees

(Do not make changes in this section until asked to do so)  
C -  
C# -  
C++ -   
Go -  
Java -  
Javascript -  
Kotlin -  
Python -

**Comment by Onkar627** [comment]
@manan025 I can do this in Java

**Comment by Onkar627** [comment]
@manan025 Can you assign this to me under Java.

**Comment by anushkarthik05** [comment]
Can you assign me this issue under C++? @manan025

**Comment by LikhithaTadikonda** [comment]
@manan025 Please assign this to me.

**Comment by manan025** [comment]
@LikhithaTadikonda - Recursive and non recursive - C
(Please make seperate PRs for recursive and non-recursive algos)

**Comment by Onkar627** [comment]
@manan025 I am willing to do this task under Java.

**Comment by manan025** [comment]
@Onkar627 recursive or non recursive?

**Comment by jdrodriguezh** [comment]
Hello! @manan025 I would like to to the Java recursive version of this!

**Comment by manan025** [comment]
Sure.
@jdrodriguezh Java - Recursive
@Onkar627 - Non recursive - Java

**Comment by TarushS** [comment]
Please assign me to the python recursive version of this.

**Comment by Mudit018** [comment]
@manan025 I am willing to work for recursive implementation in c++

**Comment by saiteja-2731** [comment]
If anyone interested in contributing good problems from DSA in C++ from leetcode or any other platform. Check out this link: [C++ help required](https://github.com/saiteja-2731/DSA-for-Placement-Preparation/issues/2)

**Comment by jAY1jS** [comment]
### **Aim**
Objective of Script
I want to contribute to the Tower of Hanoi problem (Recursive).

### **Details**
Features of the script I will provide
I can provide the solution in Java with clean code.
Can you please assign me this issue under Hacktoberfest?
### **Programming language**

- [ ]  C
- [ ]  C++
- []  Java
- [x]  Python

**Comment by manan025** [comment]
@TarushS - Python recursive
@Mudit018 - C++ Recursive
assigned

**Comment by manan025** [comment]
@jAY1jS On the basis of first come first serve, the issue for Python Recursion has already been assigned to someone else. Let me know if you want to work non-recursive algorithm.

**Comment by jAY1jS** [comment]
yes, I am willing to do a non-recursive algorithm as well.

**Comment by manan025** [comment]
@jAY1jS - Python - non recursive assigned

**Comment by SurendarSingh** [comment]
@manan025 Tower of Hanoi Javascript Recursive

**Comment by manan025** [comment]
@SurendarSingh - JS Recursive assigned

**Comment by tm2k23** [comment]
@manan025  please remove cpp label

**Comment by manan025** [comment]
@tm2k23 cpp still unassigned for non-recursive approach so I will keep the cpp label.
Thanks though.

**Comment by nakullondhe** [comment]
Is it still taking in Assignments

**Comment by manan025** [comment]
@nakullondhe Yes for some languages. You can check unassigned language in Assignees section of #113

**Comment by nakullondhe** [comment]
Can you assign me with Golang, Recursive or non-recursive

**Comment by manan025** [comment]
@nakullondhe - Go Recursive approach (If you want to do Non recursive too, you may ask here after creating PR for recursive).

**Comment by Kshma29** [comment]
@manan025 I am willing to do non-recursive implementation in C++.

**Comment by manan025** [comment]
@Kshma29 - C++ non recursive assigned
